NativeUI.csproj

```diff
@@ -34,7 +34,7 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="CitizenFX.Core">
```

Comment:
We have a [NuGet package](https://nuget.org/package/CitizenFX.Core) now - please change to use it.